pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -34,6 +39,7 @@
 , enableShared ? !stdenv.hostPlatform.isStatic
 , enableFlight ? !stdenv.isDarwin # libnsl is not supported on darwin
 , enableS3 ? true
+, enableGcs ? true
```

Comment:
```suggestion
, enableGcs ? !stdenv.isDarwin # google-cloud-cpp is not supported on darwin
```

Reply:
Is this because of the transitive `libnsl` dependency (through `grpc`)?